src/Compilers/CSharp/Portable/Binder/PatternExplainer.cs

```diff
@@ -96,6 +98,80 @@ int distance(BoundDecisionDagNode x)
             return result.ToImmutableAndFree();
         }
 
+        /// <summary>
+        /// Enumerate the paths from the root node to the node of interest, and invokes the handler
```

Comment:
>Enumerate the paths from the root node to the node of interest, and invokes the handler [](http://example.com/codeflow?start=12&length=87)

There is an inconsistency between different parts of the statement - "Enumerate", but "invoke**s**".

Reply:
The comment seems fine to me. We do enumerate the paths and invoke the handler for each one.